From 5111f7a9bbf0186b3b8375adcabf58853cce381d Mon Sep 17 00:00:00 2001
From: Ralph Boehme <rb@sernet.de>
Date: Wed, 4 Jun 2014 17:26:42 +0200
Subject: [PATCH] Ontology: add indexes for nie:mimeType and nfo:fileName

Enable indexing for "nie:mimeType" and "nfo:fileName", both are
attributes that will be used in queries generated by Netatalk and
Samba.
---
 data/ontologies/30-nie.ontology | 5 +++--
 data/ontologies/33-nfo.ontology | 3 ++-
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/data/ontologies/30-nie.ontology b/data/ontologies/30-nie.ontology
index 317a702..bfc6a87 100644
--- a/data/ontologies/30-nie.ontology
+++ b/data/ontologies/30-nie.ontology
@@ -10,7 +10,7 @@
 
 nie: a tracker:Namespace, tracker:Ontology ;
 	tracker:prefix "nie" ;
-	nao:lastModified "2011-02-08T16:00:00Z" .
+	nao:lastModified "2014-06-03T13:06:00Z" .
 
 nie:DataObject a rdfs:Class ;
 	rdfs:label "Data Object" ;
@@ -73,7 +73,8 @@ nie:mimeType a rdf:Property ;
 	rdfs:comment "File Mime Type" ;
 	nrl:maxCardinality 1 ;
 	rdfs:domain nie:InformationElement ;
-	rdfs:range xsd:string .
+	rdfs:range xsd:string ;
+	tracker:indexed true .
 
 nie:byteSize a rdf:Property ;
 	rdfs:label "Size" ;
diff --git a/data/ontologies/33-nfo.ontology b/data/ontologies/33-nfo.ontology
index ac4155f..0c157fe 100644
--- a/data/ontologies/33-nfo.ontology
+++ b/data/ontologies/33-nfo.ontology
@@ -12,7 +12,7 @@
 
 nfo: a tracker:Namespace, tracker:Ontology ;
 	tracker:prefix "nfo" ;
-	nao:lastModified "2014-01-03T02:14:35Z" .
+	nao:lastModified "2014-06-03T13:05:00Z" .
 
 nfo:Document a rdfs:Class ;
 	rdfs:label "Document" ;
@@ -500,6 +500,7 @@ nfo:fileName a rdf:Property ;
 	nrl:maxCardinality 1 ;
 	rdfs:domain nfo:FileDataObject ;
 	rdfs:range xsd:string ;
+	tracker:indexed true ;
 	tracker:fulltextIndexed true ;
 	tracker:weight 7 .
 
-- 
1.9.3

